test/optimus/scientific/ScientificTests.wdl

```diff
@@ -1,26 +1,70 @@
+task TestBam {
+
+  # Compare the tagged/aligned bam generated by Optimus to expected values derived from running Cell Ranger on the same data"
```

Comment:
I think wdl has a `description` block you can put this in

Reply:
Ah, which is indecipherably different from meta, which I already tried? 

**sigh**.